### PR TITLE
Resolves #691 document bibliography xreflabel

### LINKED ISSUE #691: Document xref label for bibliography
Ref
https://github.com/asciidoctor/asciidoctor/issues/1562

### PR DESCRIPTION
**Pull request by rockyallen**

### CODE REVIEW

File: docs/_includes/biblio.adoc [hunk 1]
```diff
@@ -4,11 +4,49 @@ Included in:
...
 include::ex-biblio.adoc[tags=base]
 ----
 
+Renders as:
+
 ====
 include::ex-biblio.adoc[tags=base]
 ====
+
+TIP: If you want advanced features such as auto-numbering and selectable citation styles, try the link:https://github.com/asciidoctor/asciidoctor-bibtex[asciidoctor-bibtex] project.
```
Comment (mojavelinux) [inline docs/_includes/biblio.adoc]: FYI the `link:` prefix is implied when linking to a fully-qualified URL. So the macro can start with `https:`.

File: docs/_includes/biblio.adoc [hunk 2]
```diff
@@ -30,14 +30,15 @@ Should it recommend putting the label before the title *and* the list as for the
 
 .For DocBook users only
 ****
+This:
```
Comment (mojavelinux) [inline docs/_includes/biblio.adoc]: How about "The following AsciiDoc fragment"

File: docs/_includes/biblio.adoc [hunk 3]
```diff
@@ -47,6 +48,6 @@ creates a bibliography element with its title set to "References", but
 ----
 
 creates a level 2 section with its title set to "References" that contains an untitled bibliography.
-You need to aware of the difference when you are customising stylesheets.
+You need to be aware of the difference when you are customising stylesheets.
```
Comment (mojavelinux) [inline docs/_includes/biblio.adoc]: Can you also change customising to the American spelling (customizing)?

**Comment by mojavelinux** [comment]
I made some updates. The reason you were confused about the different behavior is because you were missing the bibliography style on the enclosing section. The bibliography must be used in a bibliography section, which I've emphasized.

**Comment by mojavelinux** [comment]
Note that the example isn't going to work until the site is upgraded to Asciidoctor 1.5.6.1.

**Comment by rockyallen** [comment]
Thanks, much clearer.

**Pull request merged**